Create auction orders table (#2748)
cowprotocol/services: database/README.md

- PRIMARY KEY: btree(`block_number`, `log_index`)
- trade\_order\_uid: btree (`order_uid`, `block_number`, `log_index`)

### auction\_orders

> Stores all orders that were included in every Auction. A single order can be included in multiple auctions.

Reviewer: ```suggestion
Stores all orders that were included in a given auction. The same order can be included in multiple auctions.
```

 Column     | Type    | Nullable | Details
------------|---------|----------|--------

> auction\_id | bigint  | not null | in which auction this price was provided

Reviewer 2: why does it have a `\` before the `_`? the line below doesn't have it :thinking:

Author: added \ 👍

> auction\_id | bigint  | not null | in which auction this price was provided

Reviewer: ```suggestion
auction\_id | bigint  | not null | which auction this order was part of
```
